src/config/config.oc-instances.js.template

```diff
@@ -1,6 +0,0 @@
-module.exports = [
-  {
-    protocol: 'https',
-    domain: 'develop.opencast.org'
-  }
-]
```

Comment:
I though it would a nice idea to have `develop.opencast.org` as an default content provider.
Whats your opinion on this?